src/components/ActionDot.icon.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as React from 'react'
+import Svg, { G, Path, Defs, ClipPath } from 'react-native-svg'
+
+export const ActionDot = () => (
```

Comment:
Why don't you use `width`, `height`, `fill` as props like in the others?

Also, these default props values could be in a constants file inside the icons folder.

Reply:
This icon doesn't have any variation on design and I believe it will not have in the future either.

Reply:
Regarding the default prop values, I can change.

Reply:
If the others are ready for any size/color change, I don't see any reasons for us not to prepare this one also.

Reply:
I did it: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/453/commits/e51b92f9b1566251bb04436e8b364b17ce826f40